ProcessMaker/Services/ScriptMicroserviceService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ProcessMaker\Services;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use ProcessMaker\Events\ScriptResponseEvent;
+use ProcessMaker\Jobs\CompleteActivity;
+use ProcessMaker\Models\Process as Definitions;
+use ProcessMaker\Models\ProcessRequest;
+use ProcessMaker\Models\ProcessRequestToken;
+use ProcessMaker\Models\Script;
+use ProcessMaker\Models\User;
+
+class ScriptMicroserviceService
+{
+    public function handle(Request $request)
+    {
+        $response = $request->all();
```

Comment:
@gusys I can't see to get exception messages back. If my script has
```
throw new \Exception("Foo");
```
I just get
```{
  "status": "error",
  "metadata": {
    "nonce": "0.o8080cgh3c9",
    "script_id": 24,
    "instance": "http://processmaker-a.test",
    "user_id": 1,
    "user_email": "admin@processmaker.com"
  },
  "output": [],
  "message": "Script execution failed with exit code: 1"
}
```
in the scripts editor which doesn't tell me about the exception.

Reply:
@epdl indicates this has been resolved in the microservice.